contentEditableDisabled.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<meta charset="utf-8">
+<meta http-equiv="refresh" content="0; url=https://w3c.github.io/editing/docs/contentEditableDisabled.html">
```

Comment:
I'd suggest doing this instead for each of them: 

```suggestion
<script>
    let currentHash = window.location.hash;
    let newLocation = "https://w3c.github.io/editing/docs/contentEditableDisabled.html" + currentHash;
    window.location = newLocation;
</script>
```

Reply:
@johanneswilm feel free to commit this suggestion... you will need to update the other files too.